clang/test/Sema/alias-unused.c

```diff
@@ -1,7 +1,43 @@
-// RUN: %clang_cc1 -triple x86_64-linux-gnu -Wunneeded-internal-declaration -x c -verify %s
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -Wunused -x c -verify %s
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -Wunused -x c++ -verify %s
```

Comment:
I'd like to see a non-Itanium RUN line as well. Perhaps we should be using `-triple %itanium_abi_triple` and `-triple %ms_abi_triple` rather than a specific target?